test/e2e/mcd_test.go

```diff
@@ -158,6 +158,13 @@ func TestKernelType(t *testing.T) {
 
 	unlabelFunc := labelRandomNodeFromPool(t, cs, "worker", "node-role.kubernetes.io/infra")
 
+	infraNode := getSingleNodeByRole(t, cs, "infra")
+
+	osId, osVariant := getOsReleaseFromNode(t, cs, infraNode)
+	if osId == "fedora" && osVariant == "coreos" {
```

Comment:
Isn't there a convenient cluster-wide way to check for OKD (maybe the `clusterversion` CRD?) instead of querying the node to see if it's FCOS?  (We don't support joining FCOS workers to an OCP cluster AFAIK right?)